src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    public void doWork() {
+        System.out.println("Bulldozer has started working.");
```

Comment:
The message should indicate the specific type of `Machine` that is working. Consider including the class name in the message to make it more informative.